### PR TITLE
Docs: Additional improvements for the api docs

### PR DESCRIPTION
**Pull request by JeanMeche**
See individual commits.

### CODE REVIEW

File: adev/shared-docs/pipeline/api-gen/rendering/templates/header-api.tsx [hunk 1]
```diff
@@ -74,12 +74,12 @@ export function HeaderApi(props: {entry: DocEntryRenderable; showFullDescription
         )}
       </div>
 
-      <p
+      <section
```
Comment (JeanMeche) [inline adev/shared-docs/pipeline/api-gen/rendering/templates/header-api.tsx:77]: This was an issue because nested paragraph blocks are invalid in HTML.

Comment (bencodezen) [reply adev/shared-docs/pipeline/api-gen/rendering/templates/header-api.tsx:77]: Good to know!

**Review by bencodezen — APPROVED** [review]
Nice. LGTM! Going to wait for @crisbeto's before marking it for merge.

**Review by crisbeto — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 3bafda20c30edb42e5cb3c2424edca2f7224ffde.

The changes were merged into the following branches: main, 18.2.x

**Pull request closed without merging**